_posts/2015-07-23-tutorial-vagrant.markdown

```diff
@@ -0,0 +1,340 @@
+[<img alt='Imagine pagină de descărcare' src='{{ site.baseurl }}assets/images/vagrant/macos/download.png' class="noshadow" />]({{ site.baseurl }}assets/images/vagrant/macos/download.png)
+
+După ce imaginea este descărcată, dacă nu se montează automat, faceți `dublu-click` pe fișier pentru a îl monta. După ce imaginea este montată, se va deschide o fereastră _Finder_ cu conținutul, ce va arătă asemănător cu imaginea de mai jos:
+[<img alt='Imagine conținut imagine' src='{{ site.baseurl }}assets/images/vagrant/macos/image.png' class="noshadow" />]({{ site.baseurl }}assets/images/vagrant/macos/image.png)
+
+În acest moment, va trebui executat fișierul `Vagrant.pkg` pentru a iniția instalarea. Instalarea urmează un fir standard de execuție, necesitând apăsarea butonului de continuare de două ori, oferind opțiunea de a schimba locația instalării. La sfârșitul acestui proces, utilizatorul va fi solicitat să introducă parola sa pentru a putea efectua instalarea. Procesul este ilustrat în imaginile de mai jos:
+
+[<img alt='Imagine prim pas al instalării' src='{{ site.baseurl }}assets/images/vagrant/macos/firststep.png' class="noshadow" />]({{ site.baseurl }}assets/images/vagrant/macos/firststep.png)
+[<img alt='Imagine alegere a locației instalării' src='{{ site.baseurl }}assets/images/vagrant/macos/installlocation.png' class="noshadow" />]({{ site.baseurl }}assets/images/vagrant/macos/installlocation.png)
+[<img alt='Imagine solicitare a parolei' src='{{ site.baseurl }}assets/images/vagrant/macos/pass.png' class="noshadow" />]({{ site.baseurl }}assets/images/vagrant/macos/pass.png)
+
+Instalarea va continua până la sfârșit, prezentând următoarea fereastră:
+
+[<img alt='Imagine terminare instalare' src='{{ site.baseurl }}assets/images/vagrant/macos/installcomplete.png' class="noshadow" />]({{ site.baseurl }}assets/images/vagrant/macos/installcomplete.png)
+
+În acest moment este recomandată demontarea imaginii, din moment ce instalarea este finalizată. Demontarea se realizează prin apăsarea butonului indicat în imaginea de mai jos.
+
+[<img alt='Imagine demontare imagine' src='{{ site.baseurl }}assets/images/vagrant/macos/unmount.png' class="noshadow" />]({{ site.baseurl }}assets/images/vagrant/macos/unmount.png)
+
+Din acest moment, utilizarea programului command line `vagrant` va fi posibil folosind aplicația `Terminal.app` (sau `iTerm2.app` sau orice alt emulator terminal preferat). Exemplu de funcționalitate este prezentat mai jos:
+
+[<img alt='Imagine execuție vagrant help' src='{{ site.baseurl }}assets/images/vagrant/macos/vagranthelp.png' class="noshadow" />]({{ site.baseurl }}assets/images/vagrant/macos/vagranthelp.png)
+
+### Windows
+
+**Pas 1**
+
+- Instalați VirtualBox, versiunea pentru Windows! Se poate descărca de aici: [adresa de descărcare](https://www.virtualbox.org/wiki/Downloads)
+
+
+
+**Pas 2**
+
+- Vagrant are nevoie de un client SSH, iar Windows 7/8 nu îl contine by default, așa că vom folosi Git.
```

Comment:
O să vorbesc mâine (azi, de fapt) cu Elena despre asta (și mie mi s-a părut ciudat, dar în fine). Eu nici nu am un windows cu care să mă joc, deci o să îi cer ei să modifice puțin.